authors/apps/authentication/views.py

```diff
@@ -24,9 +32,72 @@ def post(self, request):
         # your own work later on. Get familiar with it.
         serializer = self.serializer_class(data=user)
         serializer.is_valid(raise_exception=True)
+
+        email = serializer.validated_data.get('email', None)
+        username = serializer.validated_data.get('username')
+
+        payload = {
+            'email': email,
+            'username': username,
+            'exp': datetime.utcnow()
+            + timedelta(minutes=60)
+        }
+        token = jwt.encode(payload, settings.SECRET_KEY,
+                           algorithm='HS256').decode('utf-8')
+        sender = os.getenv('EMAIL_HOST_USER')
+        site_link = get_current_site(request)
+        verification_link = 'http://' + site_link.domain + \
```

Comment:
If you want to use `f` strings, try and be consistent especially in one statement.